prow/cmd/deck/main.go

```diff
@@ -148,28 +148,44 @@ func staticHandlerFromDir(dir string) http.Handler {
 var (
 	deckMetrics = struct {
 		httpRequestDuration *prometheus.HistogramVec
+		httpResponseSize    *prometheus.HistogramVec
 	}{
 		httpRequestDuration: prometheus.NewHistogramVec(
 			prometheus.HistogramOpts{
-				Name:    "deck_http_request_duration_seconds",
-				Help:    "http request duration seconds for deck server",
+				Name:    "http_request_duration_seconds",
+				Help:    "http request duration in seconds",
 				Buckets: []float64{0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10},
 			},
 			[]string{"path", "method", "status"},
```

Comment:
Do we need to add a component field now that this is not specific to `deck`?

Reply:
No, Prometheus will add a `job` label to it when it scrapes